models/meshmodel/core/policies/data_models.go

```diff
@@ -17,3 +17,5 @@ type NetworkPolicyRegoResponse struct {
 	ServicePodRelationships        []RelationObject `json:"service_pod_relationships,omitempty"`
 	ServiceDeploymentRelationships []RelationObject `json:"service_deployment_relationships,omitempty"`
 }
+
```

Comment:
@MUzairS15 why do we have relationship-specific structs here?

Reply:
How will the project scale this way?

Reply:
We don't use this anymore (which was added by Abhi Kumar).
This is leftover to be deleted.

Reply:
deleted